docs/source/markdown/options/network.md

```diff
@@ -46,7 +46,7 @@ Valid _mode_ values are:
 
 - **pasta[:OPTIONS,...]**: use **pasta**(1) to create a user-mode networking
     stack. \
-    This is only supported in rootless mode. \
+    This is the default for rootless containers. \
```

Comment:
This not a equivalent change, saying this is the default is fine but you removed that it is not supported in rootful mode here which is still true